src/helpers/helpers.dom.js

```diff
@@ -124,7 +124,7 @@ export function getRelativePosition(evt, chart) {
 export function getMaximumWidth(domNode) {
 	var container = _getParentNode(domNode);
 	if (!container) {
-		return domNode.clientWidth;
+		return domNode.clientWidth || domNode.width;
```

Comment:
Test failures probably come from these changes. Need to detect if the canvas is offscreen somehow.
https://developer.mozilla.org/en-US/docs/Web/API/HTMLCanvasElement/width

Off topic, A note from MDN:
> Element.clientWidth
> Note: This property will round the value to an integer. If you need a fractional value, use element.getBoundingClientRect().

This rounding could be another source for some resize issues.

Reply:
Thanks, this was the problem!